src/procedures/procedures.11tydata.yaml

```diff
@@ -286,6 +286,189 @@ procedureMigrateZigbeeZHAOnly:
       image: z2m-backup-restore-05.png
       content: |
         <ul>
-        <li class="info">All migrated devices have a new entity ID. This means you will need to manually reconfigure automations and dashboard cards.</li>
+        <li class="info">All migrated devices have new entity IDs. This means you will need to manually reconfigure automations and dashboard cards.</li>
           <li class="info">The names that you manually assigned to your devices won't have been migrated. You will have to rename the devices again.</li>
         </ul>
+procedureMigrateConBee:
+  title: Migrate a deCONZ network to SkyConnect using ZHA
+  steps:
+    - title: Update the ConBee stick
+      image: 
+      content: |
+        <ul>
+          <li>If you are using a ConBee stick, it is recommended to have it running firmware version 26720700.</li>
+          <li>If it is not running this version, follow the procedure <b>Update the ConBee stick</b> at the end of this page.</li>
+        </ul>
+    - title: Stop the deCONZ add-on
+      image: conbee-migrate-zha-02.png
+      content: |
+        <ul>
+          <li>Under <b>Settings</b>><b>Add-ons</b>, in the <b>Zigbee2MQTT</b> add-on, select <b>Stop</b>.</li>
+        </ul>
+    - title: Remove the deCONZ integration
+      image: conbee-migrate-zha-03.png
+      content: |
+        <ul>
+          <li>Under <a class="my" href="https://my.home-assistant.io/redirect/integrations/" target="_blank"><b>Settings</b>><b>Devices & Services</b></a>, in the deCONZ integration, select <b>Delete</b>.</li>
+        </ul>
+    - title: Add a new ZHA integration
+      image: conbee-migrate-zha-04.png
+      content: |
+        <ol>
+          <li>Select <b>Add integration</b>, then select <b>ZHA</b> from the list.</li>
+          <li>Once ConBee II has been auto-detected, select <b>Configure</b>.</li>
```

Comment:
Lets add a note here that if the Discovered card is missing the user should adjust the filter settings in the overflow menu on the top right.